datasets/batch_view/main_summary/reference.md

```diff
@@ -92,6 +93,8 @@ root
  |-- env_build_version: string (nullable = true)
  |-- env_build_arch: string (nullable = true)
  |-- e10s_enabled: boolean (nullable = true)
+ |-- e10s_cohort: string (nullable = true)
```

Comment:
Are we really adding this field back?

Reply:
Heh, I think it re-appeared due to schema evolution. It certainly doesn't need to be in the docs though.